src/lib/dev-environment/dev-environment-cli.js

```diff
@@ -289,16 +289,13 @@ export async function promptForArguments( preselectedOptions: InstanceOptions, d
 	}
 
 	let multisiteText = 'Multisite';
-	let multisiteDefault = DEV_ENVIRONMENT_DEFAULTS.multisite;
-
 	if ( defaultOptions.title ) {
 		multisiteText += ` (${ defaultOptions.title } ${ defaultOptions.multisite ? 'IS' : 'is NOT' } multisite)`;
-		multisiteDefault = defaultOptions.multisite;
 	}
 
 	const instanceData: InstanceData = {
 		wpTitle: preselectedOptions.title || await promptForText( 'WordPress site title', defaultOptions.title || DEV_ENVIRONMENT_DEFAULTS.title ),
-		multisite: preselectedOptions.multisite !== undefined ? preselectedOptions.multisite : await promptForBoolean( multisiteText, !! multisiteDefault ),
+		multisite: preselectedOptions.multisite !== undefined ? resolveMultisite( preselectedOptions.multisite ) : await resolveMultisite( promptForTextBoolean( multisiteText, defaultOptions.multisite || DEV_ENVIRONMENT_DEFAULTS.multisite ) ),
```

Comment:
Are you sure you have `await` in the proper place?

```js
? resolveMultisite(/**/) : await resolveMultisite(/**/),
```

We should have it either in both cases or nowhere.

I suspect that you meant this instead:
```suggestion
		multisite: preselectedOptions.multisite !== undefined ? resolveMultisite( preselectedOptions.multisite ) : resolveMultisite( await promptForTextBoolean( multisiteText, defaultOptions.multisite || DEV_ENVIRONMENT_DEFAULTS.multisite ) ),
```

Reply:
Also please note that ESLint complains:
> This line has a length of 243. Maximum allowed is 200

PS: if you are developing in VSCode, you may find this extension useful: [dbaeumer.vscode-eslint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint).

Reply:
Oh yes! That is what I meant